packages/react-components/react-popover/src/components/Popover/Popover.test.tsx

```diff
@@ -18,6 +18,12 @@ describe('Popover', () => {
       // Popover does not have own styles
       'make-styles-overrides-win',
     ],
+    testOptions: {
+      'consistent-callback-args': {
+        // Popover onOpenChange uses Event type due to scroll event in useOnScrollOutside
```

Comment:
why does this test fail because of the `Event` type?

Reply:
This test doesn't allow Event or React.SyntheticEvent as type for event callbacks. It is asking for a more precise type

Reply:
https://github.com/microsoft/fluentui/blob/82c166c66e25d27a4b2658f576ba019283b193c7/packages/react-conformance/src/utils/validateCallbackArguments.ts#L16C5-L16C5